apiserver/facades/client/charms/client.go

```diff
@@ -664,7 +664,7 @@ func (a *API) checkCharmPlacement(arg params.ApplicationCharmPlacement) (params.
 	}
 
 	if arches.Size() > 1 {
-		// It is expected that charmhub charms are in a heterogeneous workload,
+		// It is expected that charmhub charms form a heterogeneous workload,
```

Comment:
The comment before was about the __heterogenous__ bit. Shouldn't the expectation be that this must be a **homogeneous** workload?